README.md

```diff
@@ -16,28 +16,146 @@ This simple implementation stores bubbles as directories on the server's local f
 
 ## Dependencies
 
-Requires access to an EVM-compatible blockchain via either a Web3 api service such as [Infura](https://www.infura.io/), a local blockchain node or a local [Ganache](https://trufflesuite.com/ganache/) instance.  See Configuration.
+Requires access to an EVM-compatible blockchain via either a Web3 api service such as [Infura](https://www.infura.io/), a local blockchain node or a local [Ganache](https://trufflesuite.com/ganache/) instance. Setup your Infura api or local Ganache.
 
 
-## Configuration
+# Bubble Server Configuration
```

Comment:
Should probably say 'Bubble Server Installation' since subheadings include installing dependencies and starting the server